packages/clay-date-picker/src/DayNumber.tsx

```diff
@@ -43,6 +43,13 @@ const ClayDatePickerDayNumber = ({
 
 	return (
 		<div
+			aria-selected={
+				(startDate.toDateString() !== endDate.toDateString() &&
```

Comment:
This check is duplicated at 57... what do you think about adding a const for this boolean like you have for `hasStartDateSelected`?

Reply:
Oh yes, that would be better.